Makefile.one

```diff
@@ -31,24 +32,73 @@ CMD_CAT ?= cat
 CMD_MD5 ?= md5sum
 CMD_OPA ?= opa # https://github.com/open-policy-agent/opa/releases/download/v0.33.1/opa_linux_amd64
 
-check_%:
-	@command -v $* >/dev/null || (echo "missing required tool $*" ; exit 1)
+.check_%:
+#
+	@command -v $* >/dev/null
+	if [ $$? -ne 0 ]; then
+		echo "missing required tool $*"
+		exit 1
+	else
+		touch $@ # avoid target rebuilds due to inexistent file
+	fi
```

Comment:
I'm not sure I understand why this is required.
If non-docker make is chosen, does this mean that we create these files on the system of the user?

Reply:
Those files are created during Makefile execution in current PWD. Without this files, each time the build target is triggered it will be executed. The files mean that the check was done and passed (or else they wouldn't exist). This is a well known workaround for Makefile if you need a logic to be triggered as a check, then you have to bind the logic to the existence of a file that is unmodified between runs. Or else each time you call "make -f Makefile.one tracee-ebpf" the build command will be executed (even if there are no modified files).